WhatsMac/AppDelegate.m

```diff
@@ -349,10 +352,34 @@ - (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center shouldPresentN
 }
 
 - (void)userNotificationCenter:(NSUserNotificationCenter *)center didActivateNotification:(NSUserNotification *)notification {
-    [self.webView evaluateJavaScript:
-     [NSString stringWithFormat:@"openChat(\"%@\")", notification.identifier]
-        completionHandler:nil];
-    [center removeDeliveredNotification:notification];
+    if (notification.activationType == NSUserNotificationActivationTypeReplied){
+        NSString* userResponse = notification.response.string;
+        
+        NSLog(@"Risposta Quick Reply: %@", userResponse);
+        
+        //POSTING RESPONSE
```

Comment:
Reply is sent directly from banner without "activating" the application. The message is sent silently in the background.